recipes/libsvm/all/CMakeLists.txt

```
@@ -1,17 +1,17 @@
 cmake_minimum_required(VERSION 3.4)
 project(svm C CXX)
 
-add_library(svm source_subfolder/svm.cpp)
+add_library(svm ${LIBSVM_SRC_DIR}/svm.cpp)
 set_property(TARGET svm PROPERTY WINDOWS_EXPORT_ALL_SYMBOLS ON)
-set_target_properties(svm PROPERTIES PUBLIC_HEADER source_subfolder/svm.h)
+set_target_properties(svm PROPERTIES PUBLIC_HEADER ${LIBSVM_SRC_DIR}/svm.h)
 
-add_executable(svm-predict source_subfolder/svm-predict.c)
+add_executable(svm-predict ${LIBSVM_SRC_DIR}/svm-predict.c)
 target_link_libraries(svm-predict svm)
 
-add_executable(svm-train source_subfolder/svm-train.c)
+add_executable(svm-train ${LIBSVM_SRC_DIR}/svm-train.c)
 target_link_libraries(svm-train svm)
 
-add_executable(svm-scale source_subfolder/svm-scale.c)
+add_executable(svm-scale ${LIBSVM_SRC_DIR}/svm-scale.c)
 
 install(TARGETS svm
```

Oh yeah, make sure to include `GNUInstallDirs`.

```suggestion
include(GNUInstallDirs)
install(TARGETS svm
```